server/container/guice/queue/rabbitmq/src/main/java/org/apache/james/modules/queue/rabbitmq/RabbitMQModule.java

```diff
@@ -71,31 +65,6 @@ protected void configure() {
         healthCheckMultiBinder.addBinding().to(RabbitMQMailQueueDeadLetterQueueHealthCheck.class);
```

Comment:
IMO `RabbitMQMailQueueDeadLetterQueueHealthCheck` guice binding should be in `RabbitMQMailQueueModule` instead.